### ISSUE TITLE
Support for rounding to 2 decimal places

### ISSUE THREAD

**Issue by kalbmj** [schibsted/jslt#73]
Hello,

Is it possible to round a float to 2 decimal places:

Ex. 
2.0101 -> 2.01
2.982 -> 2.98

I see examples in functions for rounding to integer, but not for a certain number of decimal places like needed.
Thanks in advance.

**Comment by larsga** [comment]
I assume that here you are talking about conversion from float to string, right? So you have `2.0101`, and `string(2.0101)` doesn't give you what you want?

**Comment by kalbmj** [comment]
Thanks for the quick response @larsga . I'm talking about rounding the number with more than 2 decimal places to one with just 2 decimal places (after that, can convert back to string)

**Comment by larsga** [comment]
So you really want to round the number as a number? Can you show me some use case for this? That is, some kind of situation that you want to use this functionality in? I know this question can seem dumb, but I'm trying to get a sense of the bigger picture before we add more functionality to the language. There are many ways this could be done, and probably quite a few of those are not what you are looking for.

**Comment by kalbmj** [comment]
Yes, seems like it's probably related more to this too: https://github.com/schibsted/jslt/issues/68

**Comment by larsga** [comment]
I'm sorry, but that doesn't help me much. Can you try to show me what you're trying to do? I don't really understand what you're asking for, just like I don't know what they're asking for in #68, either.

**Comment by kalbmj** [comment]
I'm trying to compute an average and then round that to 2 decimal places. 
So for example, something like:

{ "a": "1.001",
  "b": "1.002" } -> { "c" : round(1.0015, 2) } -> {"c" : "1.00"}

{ "a": "1.091",
  "b": "1.099" } -> { "c" : round(1.095, 2) } -> {"c" : "1.10"}

**Comment by larsga** [comment]
Thinking about this some more I think the right thing to do here is what you wanted all along: have `round` support an optional number of digits. I'll try to implement that soon.

**Comment by larsga** [comment]
Let me double-check something here.

Which of these is it you want?

Input: `{"a" : 1.0015, "b" : 1.095}`

Output: `{"a" : "1.00", "b" : "1.10"}`

Or: `{"a" : 1.0, "b" : 1.1}`

I'm asking because if you do: `{for(.) .key : string(round(.value, 2))}` then you will actually get `{"a" : "1.0", "b" : "1.1"}` because decimal numbers don't have any specific number of digits.
